plugins/vppv2/aclplugin/descriptor/acl.go

```diff
@@ -99,14 +101,68 @@ func (d *ACLDescriptor) EquivalentACLs(key string, oldACL, newACL *acl.Acl) bool
 		return false
 	}
 	for i := 0; i < len(oldACL.Rules); i++ {
-		if !proto.Equal(oldACL.Rules[i], newACL.Rules[i]) {
+		if !d.equivalentACLRules(oldACL.Rules[i], newACL.Rules[i]) {
 			return false
 		}
 	}
 
 	return true
 }
 
+// equivalentACLRules compares two ACL rules, handling the cases of unspecified
+// source/destination networks.
+func (d *ACLDescriptor) equivalentACLRules(rule1, rule2 *acl.Acl_Rule) bool {
+	if rule1.Action != rule2.Action || !proto.Equal(rule1.MacipRule, rule2.MacipRule) {
```

Comment:
For better code clarity I would suggest to split checks like this which don't have much in common and rather group them by specific sections/fields.